couchbase/json-buckets.yml

```diff
@@ -37,18 +37,18 @@ observation:
         type: long_gauge
         label: bytes read
         unit: bytes
-        description: Bytes read
+        description: Average background wait time
 
       - name: bytes.read
         source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.bytes_read[0]
-        type: long_gauge
+        type: double_gauge
```

Comment:
This one is probably long, not double. Same for `written`.

Reply:
Also possibly a counter.

Reply:
It is actually double, i check some real metrics, it return some real value.

Reply:
How can bytes.read be a decimal value? Do they report some kind of average or rate here? If yes, then the metric is not very useful, but in any cases needs different name and description.

Reply:
Update name